content/docs/gto/get-started-dvc.md

```diff
@@ -0,0 +1,176 @@
+# Get Started DVC
+
+To leverage concepts of Model and Data Registries in a more explicit way, you
+can denote the `type` of each output. This will let you browse models and data
+separately, address them by `name` in `dvc get`, and eventually, see them in DVC
+Studio.
+
+Let's start with marking an artifact as data or model.
+
+If you're using `dvc add` to track your artifact, you'll need to run:
+
+```dvc
+# note that all CLI options are optional:
+$ dvc add models/mymodel.pkl \
+    --type model \  # this makes DVC understand it's an ML model
+    --name def-detector \
+    --description "glass defect image classifier" \
+    --label "algo=cnn" \
+    --label "owner=aguschin" \
+    --label "project=prod-qual-002"
+```
+
+<details>
+
+### Beside tracking this as usually, his will add it to a top section called `registry` in your `dvc.yaml`
+
+```yaml
+# dvc.yaml
+registry:
+  def-detector: # just like with plots, this could be a path or any string ID
+    type: model
+    description: glass defect image classifier
+    labels:
+      - algo=cnn
+      - owner=aguschin
+      - project=prod-qual-002
+    path: models/mymodel.pkl # specify path if use alias to name this
+```
+
+If you want this to be in a separate file (say, `artifacts.yaml`), you can tell
+DVC to use it with:
+
+```yaml
+# dvc.yaml
+registry: artifacts.yaml
+```
```

Comment:
Extra for now, but was requested by some users

Reply:
Love this!  I would love to do this with `plots` too.